notebooks/official/prediction/pytorch_image_classification_with_prebuilt_serving_containers.ipynb

```diff
@@ -0,0 +1,883 @@
+{
```

Comment:
<p>For consistency, the header should be "Get batch predictions" instead of "Make".</p><p> </p><p>For the batch input, I would link them to the docs for more details. Perhaps something like:</p><p> </p><p> </p><p>You create a batch input file in JSONL format and store the input file in your Cloud Storage bucket. For more information, see [Input data requirements](https://cloud.google.com/vertex-ai/docs/predictions/get-predictions#input_data_requirements).</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1592/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>